packages/manager/modules/octavia-load-balancer/src/create/routing.js

```diff
@@ -0,0 +1,38 @@
+import { TRACKING_NAME, TRACKING_CHAPTER_1 } from './constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('octavia-load-balancer.create', {
+    url: '/create',
+    component: 'octaviaLoadBalancerCreate',
+    atInternet: {
+      rename: TRACKING_NAME,
+    },
+    resolve: {
+      serviceName: /* @ngInject */ ($transition$) =>
+        $transition$.params().serviceName,
+      breadcrumb: /* @ngInject */ ($translate) =>
+        $translate.instant('octavia_load_balancer_create_title'),
+      user: /* @ngInject */ (coreConfig) => coreConfig.getUser(),
```

Comment:
**nitpick:** Can be directly injected in Controller since it is a synchronous call